microsoft-edge/webview2/how-to/winrt-from-js.md

```diff
@@ -0,0 +1,421 @@
+---
+title: Call native-side WinRT code from web-side code
+description: How to pass WinRT host objects to JavaScript using the AddHostObjectToScript API for WebView2 apps.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.technology: webview
+ms.date: 05/03/2022
+---
+# Call native-side WinRT code from web-side code
+
+Your web-side JavaScript code can access native-side WinRT methods and properties, with the help of the **wv2winrt** tool (the WebView2 WinRT JS Projection tool).  The **wv2winrt** tool generates needed code files for your JavaScript code, and enables using methods and properties of any WinRT APIs, including:
+
+*  Your WebView2 host app's WinRT APIs.
+*  Windows WinRT APIs.
+*  Third-party WinRT APIs.
+
+For more information about why you'd want to have your web-side JavaScript code access the methods and properties of your WinRT host app, see the introduction of [Call native-side code from web-side code](hostobject.md).
+
+This article walks you through the following main steps:
+
+<!-- instance 1 of this paragraph: -->
+1. Create a **WinRTAdapter** project for the wv2winrt tool.  This project builds a library out of code that's generated by running the tool.  This generated code allows WinRT APIs to be exposed in the WebView2 control.
```

Comment:
The two last sentences are included below, in the details for this step. It might be too much detail to have them here?